packages/block-library/src/site-logo/index.php

```diff
@@ -32,6 +32,10 @@ function render_block_core_site_logo( $attributes ) {
 		$classnames[] = "align{$attributes['align']}";
 	}
 
+	if ( !empty( $attributes['width'] ) ) {
+		$classnames[] = "is-resized";
```

Comment:
Would it be clearer if it was `is-default-size` or something instead? It would simplify the `if()` statement as you could remove the `!`. It could simplify the CSS that targets this, too, as you wouldn't need the `:not()`.

Reply:
Good thought. Created a separate PR for that here: https://github.com/WordPress/gutenberg/pull/30846